images/makeproject/config.xml

```diff
@@ -36,7 +36,7 @@
         <one_result_per_host_per_wu/>
         <max_wus_in_progress>1</max_wus_in_progress>
         <max_wus_in_progress_gpu>1</max_wus_in_progress_gpu>
-        <max_wus_to_send>1</max_wus_to_send>
+        <max_wus_to_send>8</max_wus_to_send>
```

Comment:
Increasing `max_wus_to_send` will be ineffective, given a value 'N' it uses this formula to determine the number of workunits to send: `N*(NCPUS + GM*NGPUS)`
See `max_wus_in_progress` and `max_wus_in_progress_gpu` https://boinc.berkeley.edu/trac/wiki/ProjectOptions#Joblimits